bindings/python/setup.py

```
@@ -58,6 +59,33 @@
+
+def _stage_license_files():
+    if not os.path.isfile(source_license):
+        raise RuntimeError(f"Root license file does not exist: {source_license}")
+    shutil.copy2(source_license, packaged_license)
+
+    if os.path.isdir(source_notices_dir):
+        if os.path.isdir(packaged_notices_dir):
+            shutil.rmtree(packaged_notices_dir)
+        shutil.copytree(source_notices_dir, packaged_notices_dir)
+
+def _license_files():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_license_files`

(ANN202)